### DISCUSSION TITLE
Not working with PNPM

### DISCUSSION THREAD

**Discussion by raiyansarker** [maizzle/framework#833] [General]
Hi,
I wanted to try out maizzle. So I installed the starter package and found out that pnpm is not working. Npm works, yarn works, but not pnpm. Tried different versions of Node, cleaned cache, still npm and yarn works but not pnpm. It gives this error.
<img width="821" alt="Screenshot 2022-11-10 at 12 29 55 PM" src="https://user-images.githubusercontent.com/38852396/201017362-66dac790-8221-4ed9-8f6d-cae39a12fb9b.png">

I have a monorepo that uses pnpm. At the end, I will use maizzle inside the monorepo. Is there anything that I am doing wrong?

**Comment by cill-i-am** [comment]
I'm having the same issue. added mazzle as an app in a turborepo monorepo & used pnpm. cd'ing into the directory and using npm works though

**Comment by antony-gsdts** [comment]
+1 on this. Can't resolve exactly this.

**Reply by antony-gsdts** [reply]
Installing the `tailwindcss` package fixed this for me.
